packages/edit-widgets/src/components/header/index.js

```diff
@@ -49,11 +52,15 @@ function Header( { setListViewToggleElement } ) {
 						setListViewToggleElement={ setListViewToggleElement }
 					/>
 					{ hasFixedToolbar && isLargeViewport && (
-						<Slot
-							className="selected-block-tools-wrapper"
-							name="__experimentalSelectedBlockTools"
-							bubblesVirtually
-						/>
+						<>
+							<div className="selected-block-tools-wrapper">
+								<BlockContextualToolbar isFixed />
+							</div>
+							<Popover.Slot
+								ref={ blockToolbarRef }
+								name="block-toolbar"
```

Comment:
That's interesting, if this popover need to be adjacent to the block toolbar (always), why can't we just render it within the BlockToolbar component?

Reply:
I think that's the eventual preference, but to integrate it into the `<BlockToolbar />` so it is no longer its own floating toolbar. But, for now, I didn't want to also make that happen in this PR since it seems like something that would end up blocking this work.

Reply:
Follow-up issue created with implementation plan: https://github.com/WordPress/gutenberg/issues/56047